### PR TITLE
Clarify how to create a CNAME

### LINKED ISSUE #2184: Update instructions in [Configuring a subdomain] help article
### What article on docs.github.com is affected?

https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/managing-a-custom-domain-for-your-github-pages-site#configuring-a-subdomain

### What part(s) of the article would you like to see updated?

Current:

> To set up a `www` or custom subdomain, such as `www.example.com` or `blog.example.com`, you must create a _CNAME_ file in your site's repository and configure a `CNAME` record with your DNS provider.

It should probably be changed to something like:

> To set up a `www` or custom subdomain, such as `www.example.com` or `blog.example.com`, you must add your domain in the repository settings, which will create a _CNAME_ file in your site’s repository. After that, configure a `CNAME` record with your DNS provider.

### Additional information

Users should NOT create the CNAME file themselves, this is done automatically once they update their custom domain settings with the domain name and hence the need for this change.

If we can get the wording that would explicitly convey to users that they should NOT create the CNAME file, that will be even better

 CC for visibility
@github/support-technical
@github/business-support-all

### PR DESCRIPTION
**Pull request by ajsirota**
Clarify that users should create a CNAME in repo settings, rather than creating it directly. 

### Why:
Close issue #2184.

### What's being changed:
See the diff above

### Check off the following:
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Review by lucalves — CHANGES_REQUESTED** [review]
Why remove code tags?

**Comment by ajs256** [comment]
Hmm. Don't know how that happened. It's fixed now!

**Comment by janiceilene** [comment]
Thanks @ajs256! I'll get this triaged for review ✨

**Review by lucalves — APPROVED** [review]
Now, is ok to me!

**Review by duc811997 — COMMENTED** [review]
[Account 0.txt](https://github.com/github/docs/files/5861310/Account.0.txt)

**Review by duc811997 — APPROVED** [review]
[github-recovery-codes.txt](https://github.com/github/docs/files/5861312/github-recovery-codes.txt)

**Comment by ajs256** [comment]
Hey there @duc811997. I would strongly advise changing your recovery codes now, since you seem to have accidentally uploaded them. @janiceilene or someone else with permissions, could you please edit their comment to remove the file?

**Comment by ajs256** [comment]
Wait a minute, did I close this? I don't seem to be able to re-open it.

**Comment by janiceilene** [comment]
Re-triaged!

File: content/github/working-with-github-pages/managing-a-custom-domain-for-your-github-pages-site.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ The `dig` command, which can be used to verify correct configuration of DNS reco
 
 ### Configuring a subdomain
```
Comment (adonisj910) [inline content/github/working-with-github-pages/managing-a-custom-domain-for-your-github-pages-site.md:36]: https://GitHub.com/adonisj910.md

**Review by adonisj910 — CHANGES_REQUESTED** [review]
git commit

Request changes

File: content/github/working-with-github-pages/managing-a-custom-domain-for-your-github-pages-site.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ The `dig` command, which can be used to verify correct configuration of DNS reco
```
Comment (adonisj910) [inline content/github/working-with-github-pages/managing-a-custom-domain-for-your-github-pages-site.md:35]: https://GitHub.com/adonisj910.md

**Review by adonisj910 — COMMENTED** [review]
👍

**Comment by ajs256** [comment]
Looks like [your stalebot fix](https://github.com/github/docs/commit/f10360f689a3e8011ddc3050d636ee84fcdda096)isn't working...

**Comment by ajs256** [comment]
I can't re-open it, but I will not delete the branch so someone can re-open it and hopefully merge it.

**Comment by janiceilene** [comment]
It looks like our new action inadvertently closed this :upside_down_face: Sorry about that! Reopening and triaging this for review :yellow_heart: Thanks for not deleting the branch!

**Review by felicitymay — APPROVED** [review]
Thanks for your PR and your patience with the technical glitches.

Your change looks good to me on our staging site: https://docs-2961--patch-2.herokuapp.com/en/github/working-with-github-pages/managing-a-custom-domain-for-your-github-pages-site#configuring-a-subdomain. It clearly fixes the linked issue ✨

**Pull request merged**